package.json

```diff
@@ -40,10 +40,9 @@
   },
   "homepage": "https://github.com/Skyscanner/backpack-android#readme",
   "devDependencies": {
-    "babel-eslint": "^8.2.1",
```

Comment:
This dependency  is encapsulated inside `eslint-config-skyscanner-with-prettier`